Gruntfile.js

```diff
@@ -400,8 +391,7 @@ module.exports = function(grunt) {
     'default',
     'minify',
     'karma:unit_ci',
-    'nodeunit',
-    'exec:deployci'
+    'nodeunit'
```

Comment:
I'm not worrying about travis - making it run in CI is a separate issue. I'm just trying to get it running reliably in dev for now.